Sources/Paywalls/PaywallData.swift

```diff
@@ -15,6 +15,37 @@
 
 import Foundation
 
+/// Represents countries where currencies typically have zero decimal places
+public struct ZeroDecimalPlaceCountries: Codable, Sendable, Hashable, Equatable {
+
+    /// Storefront IDs that should typically display zero decimal places
+    public var apple: Set<String> {
+        _apple
+    }
+
+    private var google: Set<String> {
+        _google
+    }
+
+    @DefaultDecodable.EmptyArray
+    internal private(set) var _google: Set<String>
+
+    @DefaultDecodable.EmptyArray
+    internal private(set) var _apple: Set<String>
+
+    /// Storefront IDs that should typically display zero decimal places on Apple and Google devices, respectively
+    public init(apple: Set<String>, google: Set<String>) {
+        self._apple = apple
+        self._google = google
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case _apple = "apple"
+        case _google = "google"
+    }
```

Comment:
This seems to be a limitation of the property wrapper, that it can't work on public types. The code you proposed generates an error "Property cannot be declared public because its property wrapper type uses an internal type". Based on the existing code below, I figured this was how it should be used.

```
    @DefaultDecodable.Zero
    internal private(set) var _revision: Int = 0

    /// The revision identifier for this paywall.
    public var revision: Int {
        get { return self._revision }
        set { self._revision = newValue }
    }
```

Reply:
Ohhhhhh ewww 😬 I guess that makes sense then 🤷‍♂️

Reply:
@jamesrb1 That property wrapper might not even be necessary since we are doing some optional chaining and nil coalescing here - https://github.com/RevenueCat/purchases-ios/pull/4132/files#diff-8ea63d6187042a18f8814d96bafd8472d618f95d1bf67def849fae6b041b28e0R63-R65

I'd opt for just making it `public var apple: [String]` without the property wrapper I think? 🤔

Reply:
You are totally right, and done!